pkg/limits/config.go

```diff
@@ -36,6 +36,11 @@ type Config struct {
 
 	KafkaConfig kafka.Config `yaml:"-"`
 
+	// TODO(grobinson): These will be removed when we figure out how to support
+	// different addresses for read and write clients in kafka.Config,
+	KafkaReadAddress  string `yaml:"kafka_read_address"`
```

Comment:
This is a quick fix to support different addresses for read and write agents. I tried having separate configs, but it creates other problems. I need time to think about the correct solution here